app/assets/javascripts/modules/publisher-table.js

```diff
@@ -0,0 +1,48 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function PublisherTable ($module) {
```

Comment:
I wonder whether we should rename this as "PublisherTable" seems a bit generic…maybe "PublicationsTable"?